### PR TITLE
Fix dynamic imports and make sure we only load component when found

### PR DESCRIPTION
**Pull request by dorelljames**

### CODE REVIEW

File: component/sections/hero/index.js [hunk 1]
```diff
@@ -1,10 +1,18 @@
-import React from "react"
-import dynamic from "next/dynamic"
+import React from "react";
+import dynamic from "next/dynamic";
+
+const Variants = [
```
Comment (mamfi-0221) [inline component/sections/hero/index.js:4]: `Variants` is defined as a constant array, please change to object.
With current implementation, adding this section on demo page shows no preview. Please see this [loom video](https://www.loom.com/share/69c7529ff0314f24a249dd24edda3a5f), thanks!

File: component/sections/how_it_works/index.js [hunk 2]
```diff
@@ -1,17 +1,27 @@
...
-import dynamic from "next/dynamic"
+import React from "react";
+import dynamic from "next/dynamic";
+
+const Variants = {
+  variant_a: dynamic(() => import("./variant_a")),
+  variant_b: dynamic(() => import("./variant_b")),
+  variant_c: dynamic(() => import("./variant_c")),
+  variant_d: dynamic(() => import("./variant_d")),
+  variant_d: dynamic(() => import("./variant_d")),
```
Comment (mamfi-0221) [inline component/sections/how_it_works/index.js:9]: Kindly change `line 9` or the 5th variant to `variant_e`.
With current implementation, when variant_e is selected on studio, no web preview is shown as highlighted in this [video](https://www.loom.com/share/201070128a9f46fd98c6bfbc4a77c8f8).

**Review by mamfi-0221 — CHANGES_REQUESTED** [review]

File: component/sections/hero/index.js (hunk 1 above)
Comment (dorelljames) [reply component/sections/hero/index.js]: Thanks! Definitely needs to be fixed.

File: component/sections/how_it_works/index.js (hunk 2 above)
Comment (dorelljames) [reply component/sections/how_it_works/index.js]: Thanks yes!

File: component/sections/hero/index.js (hunk 1 above)
Comment (dorelljames) [reply component/sections/hero/index.js]: Fix in [`8090572` (#18)](https://github.com/webriq-pagebuilder/site-template-default/pull/18/commits/809057252ba29b2728a584b373b8cc1fa35b8486)

File: component/sections/how_it_works/index.js (hunk 2 above)
Comment (dorelljames) [reply component/sections/how_it_works/index.js]: Fix in [`8090572` (#18)](https://github.com/webriq-pagebuilder/site-template-default/pull/18/commits/809057252ba29b2728a584b373b8cc1fa35b8486)

File: component/sections/hero/index.js (hunk 1 above)
Comment (mamfi-0221) [reply component/sections/hero/index.js]: Approved. Confirming fix.

File: component/sections/how_it_works/index.js (hunk 2 above)
Comment (mamfi-0221) [reply component/sections/how_it_works/index.js]: Approved. Confirming fix.

**Pull request merged**